js/index.js

```diff
@@ -1 +1,105 @@
-// Your code goes here
+// CODE: //
+
+// ============== CONSTANTS ============ //
+
+const navLinks = document.querySelector('.nav');
+
+
+
+// =============== MOUSEOVER ============ //
+
+navLinks.addEventListener('mouseover', (e) => {
+    e.target.style.color= "#411f9e";
+    setTimeout(function() {
+        e.target.style.color = "";
+      }, 400);
+    }, false);
+
+
+// ================ KEYDOWN ============= //
+
+document.addEventListener('keydown', (e) => {
+    const keyDown = e.key;
+    console.log('Hey! I am working.')
+    alert ('Uhm, that tickled!!!');
+});
+
+
+// ================WHEEL =============== //
```

Comment:
Remember, a mouse wheel event is a lot like the scroll event. I know you can do it! :fireworks: